### ISSUE TITLE
AABB_tree demo uses 100% CPU while doing nothing

### ISSUE THREAD

**Issue by dpellegr** [CGAL/cgal#4766]
Possibly related to: https://github.com/CGAL/cgal/issues/970

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): ArchLinux 64 bits
* Compiler: GCC 10
* Release or debug mode: Release
* CGAL version: trunk (git clone)

**Comment by sloriot** [comment]
Please provide a minimal example showing the problem, otherwise we cannot help you.

**Comment by sloriot** [comment]
OK, I confirm that we cannot open a file with the demo anymore. @maxGimeno ?

**Comment by sloriot** [comment]
Note that the simplest fix is to no longer ship the demo after checking that all its functionalities are included in the polyhedron demo.

**Comment by afabri** [comment]
Well, if the fix is a one-liner, maybe provide it to @dpellegr

**Comment by lrineau** [comment]
@dpellegr If you can use a Git version of CGAL, please test PR #4770, that addresses the issue.

**Comment by dpellegr** [comment]
@lrineau Yes, I can confirm that https://github.com/CGAL/cgal/pull/4770/commits/1d4d3efcc9b0f91db8e5c962b0dc59614212b502 fixed this issue.
